src/mapport.cpp

```diff
@@ -35,9 +35,11 @@ static_assert(MINIUPNPC_API_VERSION >= 10, "miniUPnPc API version >= 10 assumed"
 static CThreadInterrupt g_upnp_interrupt;
 static std::thread g_upnp_thread;
 static constexpr auto PORT_MAPPING_REANNOUNCE_PERIOD = std::chrono::minutes(20);
+static constexpr auto PORT_MAPPING_RETRY_PERIOD = std::chrono::minutes(5);
 
-static void ThreadMapPort()
+static bool ThreadUpnp()
```

Comment:
In commit "net: Keep trying to use UPnP when -upnp=1"

This is confusingly named when it isn't actually a thread on its own anymore.

Reply:
[Renamed](https://github.com/bitcoin/bitcoin/pull/18077#issuecomment-591285675).